Source/Entities/EntityInstanceBase.generated.cst

```diff
@@ -732,10 +733,13 @@ WriteRelationshipPropertyString();
 		/// </returns>
 		public bool IsPropertyChanged(<%= GetClassName(SourceTable, ClassNameFormat.Column) %> column)
 		{
+			if (_originalData == null)

```

Comment:
Any reason for this change?

Reply:
If property tracking has not been initialized, then _originalData is null when this method is called and blows up.
I suppose another option would be to throw a custom error telling the user to initialize tracking.